ownCloudAppShared/Client/Account/Connection/AccountConnection.swift

```diff
@@ -526,6 +530,28 @@ open class AccountConnection: NSObject {
 		}
 	}
 
+	func updateServerLogo() {
```

Comment:
Right now, ´updateServerLogo()` is called via `updateConnectionStatusSummary()` whenever the connection status changes to `.online`, which could happen multiple times for a single connection. I think a simple mechanism like the one using `userAvatarUpdated` in `updateUserAvatar()` that prevents multiple downloads/resource requests during a single connection would be good to avoid unnecessary overhead.

Reply:
Fixed in latest commit